docs/tutorials/containermonitoringaddon.md

```diff
@@ -9,7 +9,8 @@ Container health gives you performance monitoring ability by collecting memory a
 
 # Prerequisites!
 
-  - A Log Analytics workspace. You can create it when you enable monitoring of your new AKS cluster or let the onboarding experience create a default workspace in the default resource group of the AKS cluster subscription. If you chose to create it yourself, you can create it through [Azure Resource Manager], through [PowerShell], or in the [Azure portal].
+  - A Log Analytics workspace. You can create it when you enable monitoring of your new AKS cluster. You can create it through [Azure Resource Manager], through [PowerShell], or in the [Azure portal].
```

Comment:
Should we change "your new AKS cluster" to "your new AKS Engine cluster"?

Reply:
removed this section so this comment not applicable now.